helpers/reviews.js

```diff
@@ -1,18 +1,361 @@
+            }, { new: true })
+        })
+        if (result)
+            return Status.createSuccessResponse(200, {
+                message: "Comment successfully added to Review."
+            })
+    } catch (err) {
+        console.error('add comment to review caught error:', err.message)
+        return Status.createErrorResponse(400, err.message)
+    }
+}
+
+const getAllComments = async (reviewId) => {
+    try {
+        let review = await getReviewById(reviewId)
+        return review.comments
+    } catch (err) {
+        console.error('get all comments caught error:', err.message)
+        return Status.createErrorResponse(400, err.message) 
+    }
+}
+
+const deleteRating = async (ratingId) => {
+    try {
+        let result = await db(MONGO_URL, () => {
+            return Rating.findOneAndDelete({
+                _id: ratingId
+            })
+        })
+        if (result) 
+            console.log(`Rating successfully DELETED: ${ratingId}`)
+            return { OKmessage: `Rating successfully DELETED: ${ratingId}` }
+    } catch (err) {
+        console.error('delete rating caught error:', err.message)
+        return Status.createErrorResponse(400, err.message)
+    }
+}
+
+const deleteAllComments = async (payload) => {
+    try {
+        let commentsList = await getAllComments(payload.review_id)
+        let result = await db(MONGO_URL, () => {
+            return Comment.deleteMany({
+                _id: {
+                    $in: commentsList //array of comments
+                }
+            })
+        })
+        if (result) 
+            return { OKmessage: "All comments successfully DELETED." }
+    } catch (err) {
+        console.error('delete all comments caught error:', err.message)
+        return Status.createErrorResponse(400, err.message)
+    }
+}
+
+//Creates a new Rating obj and saves to db. Returns rating ID
+const createRating = async (eventBody) => {
+    let payloadIsValid = await RequestChecker(eventBody, Rating)
+    if (!payloadIsValid) return "payload does not match model."
+    let newRating = Rating({
+        _id: new mongoose.Types.ObjectId(),
+        culture: eventBody.culture,
+        mentorship: eventBody.mentorship,
+        impact: eventBody.impact,
+        interview: eventBody.interview
+    })
+    try {
+        let result = await db(MONGO_URL, () => newRating.save())
+        console.log('New Rating Created:\n', result)
+        return newRating._id
+    } catch (err) {
+        console.error('caught err while trying to save Rating to db:\n', err.message)
+    }
+}
+
+//--------------- EXPORTED FUNCTIONS ---------------
+
+//013_FEAT_CRUD-REVIEW
+    //createReview 1.0
+module.exports.createReview = async (payload) => {
+    console.log('payload:\n', payload)
+    let foundUserId = await findUserId(payload)
+    let ratingId = await createRating(payload)
+    let foundCompany = await findCompanyByName(payload)
+    console.log('foundCompany:\n', foundCompany)
```

Comment:
ideally we would need to check if foundUserId, ratingId, and foundCompany are not null